docs/UserGuide.md

```diff
@@ -645,7 +646,7 @@ suggest new features and it is easily extensible!
 | **Delete a contact** | `delete INDEX`                                                                         | `delete 3`                                                  |
 | **Tag a contact**    | `tag INDEX m/MODULE...`, `tag m/MODULE...`, `tag INDEX g/GROUP`, `tag g/GROUP`         | `tag m/CS1234`, `tag m/CS2345 mon 12 1`, `tag 1 g/Friend`    |
 | **Untag a contact**  | `untag INDEX m/MODULE...`, `untag m/MODULE...`, `untag INDEX g/GROUP`, `untag g/GROUP` | `untag m/CS1234`, `untag m/CS2345 mon 12 1`, `tag 1 g/Friend` |
-| **Filter contacts**  | `find z/FIELD`                                                                         | `find n/Tan`, `find m/CS1231`                               |
+| **Filter contacts**  | `find z/FIELD... z/FIELD...`                                                           | `find n/Tan`, `find m/CS1231`                               |
```

Comment:
This one can consider putting both prefixes in the same command, `find n/Tan m/CS1231`.